### PR TITLE
Add support for syslog server hostname

### PR DESCRIPTION
**Pull request by lukebakken**
schlagert/syslog version 3.4.3 adds support for specifying a host name for `dest_host`, this PR supports it as well

See this gist for my test procedure:

https://gist.github.com/lukebakken/20bd05816a8b80839cc3c6645350bfa4

### CODE REVIEW

File: priv/schema/rabbit.schema [hunk 1]
```diff
@@ -1086,9 +1086,8 @@ end}.
     {datatype, {enum, [true, false]}}
 ]}.
 
-{mapping, "log.syslog.ip", "syslog.dest_host", [
```
Comment (michaelklishin) [inline priv/schema/rabbit.schema]: Can we leave `log.syslog.ip` in place for compatibility? (it also makes some sense as an alias anyway)

Comment (lukebakken) [reply priv/schema/rabbit.schema]: I wanted to do that, but since they both set `syslog.dest_host`, it doesn't work.

Comment (michaelklishin) [reply priv/schema/rabbit.schema]: Oh, OK.

Comment (lukebakken) [reply priv/schema/rabbit.schema]: Hm there may be options if the value can be transformed, maybe. Let me check.

**Comment by lukebakken** [comment]
@michaelklishin - ready for re-review, both `log.syslog.ip` and `log.syslog.host` are supported now :smiley_cat:

**Comment by lukebakken** [comment]
Thanks for the review @michaelklishin

**Pull request merged**